pkg/services/sqlstore/datasource_test.go

```diff
@@ -246,17 +250,18 @@ func TestDataAccess(t *testing.T) {
 			return nil
 		})
 
-		err := sqlStore.DeleteDataSource(context.Background(), &models.DeleteDataSourceCommand{ID: ds.Id, UID: "nisse-uid", Name: "nisse", OrgID: 123123})
+		err := sqlStore.DeleteDataSource(context.Background(),
```

Comment:
@MasslessParticle was it intentional to fire an event when the data source doesn't actually exist?